challenge.rb

```diff
@@ -1,26 +1,34 @@
 
 def capitalize_each_string(input)
   #implement your solution here
+  input.map(&:capitalize)
 end
 
 def fetch_the_dog(input)
   #implement your solution here
+  [input.find {|i| i == "dog"}]
 end
 
 def no_dogs_allowed(input)
   #implement your solution here
+  input.reject {|i| i == "dog"}
 end
 
 def count_the_animals(input)
   #implement your solution here
+  input.count
 end
 
 def fetch_the_first_two(input)
   #implement your solution here
+  input.first(2)
 end
 
 def fetch_CD_animals(input)
   #implement your solution here
+  input.select do |animal|
+  	animal[0] == 'c' || animal[0] == 'd'
```

Comment:
The [`String#start_with?`](https://ruby-doc.org/core-2.5.0/String.html#method-i-start_with-3F) method can simplify this a bit.